src/ui/FormWidgets/TextInput.ts

```diff
@@ -145,6 +156,17 @@ export class TextInput implements IFormWidget, IFormWidgetSettable {
     this.name && this.createLabelOrPlaceholder();
   }
 
+  private buildIcon() {
+    if (!this.options.prefixingIcon) {
+      return;
+    }
+    const iconClassName = `${this.options.className}-icon`;
+    this.icon = $$('span', { className: iconClassName });
+    this.icon.setHtml(SVGIcons.icons[this.options.prefixingIcon] || this.options.prefixingIcon);
```

Comment:
Related to the comment above, is the second half of the `||` condition necessary at this point in time?

Reply:
It's unnecessary, I'll remove it.